src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -173,7 +173,11 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
               if (this.loggedInUser) {
                 // Registration key belongs to another user who is not the logged in user
                 this.navigationService.navigateWithErrorMessage(this.router, '/web/front',
-                    'You are not authorized to view this page.');
+                    `You are trying to access TEAMMATES using the Google account ${this.loggedInUser}, which
+                    is not linked to any TEAMMATES account. If you used a different Google account to
```

Comment:
"not linked to any TEAMMATES account" is not accurate. What's happening here is the logged in user doesn't match the google ID of the student entity linked by the registration key, but the logged in user might actually be linked to some other valid students (but we don't need to find out about this).

@damithc fyi

Reply:
Good point @wkurniawan07  
Rephrase as `not linked to this TEAMMATES course`?

However, if it will not fit if the user is trying to just login to TEAMMATES (not access a specific course).